pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "numalogic"
-version = "0.6.1"
+version = "0.6.2a1"
```

Comment:
if you have tested it already make it 0.6.2, and cut a release

Reply:
got it